.github/workflows/release.yaml

```diff
@@ -0,0 +1,33 @@
+name: Release changesets
+
+# This workflow will be triggered on push to the main branch
+# on:
+#   push:
+#     branches:
+#       - main
+# temporarly manual triggers only
+on:
+  workflow_dispatch:
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js 20
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Install Dependencies
+        run: yarn install
+
+      - name: Create Release Pull Request
+        uses: changesets/action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I am pretty sure we miss some npm authorisation variables here (i.e. NPN_TOKEN),
see [with publishing](https://github.com/changesets/action?tab=readme-ov-file#with-publishing) section of "Changesets Release Action"

Reply:
I think we can do it in next PR, to test if creating PRs work first.

Reply:
Yes, that was actually my intent. Make sure that changesets are fine, then setup NPM and test NPM publish separately